src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3836,6 +3860,23 @@ input InputUnpreferred {
   useUnpreferredRoutesPenalty: Int @deprecated(reason : "Use unpreferredCost instead")
 }
 
+"Filters an entity by a date range."
+input LocalDateRangeInput {
+  """
+  **Exclusive** end date of the filter. This means that if you want a time window from Sunday to
+  Sunday, `end` must be on Monday.
+    
+  If `null` this means that no end filter is applied and all entities that are after or on `start`
+  are selected.
+  """
+  end: LocalDate
+  """
+  **Inclusive** start date of the filter. If `null` this means that no `start` filter is applied and all
+  dates that are before `end` are selected.
+  """
+  start: LocalDate
```

Comment:
Order by chronologically here, the `start` should be before the `end`:

```suggestion
  """
  **Inclusive** start date of the filter. If `null` this means that no `start` filter is applied and all
  dates that are before `end` are selected.
  """
  start: LocalDate
  """
  **Exclusive** end date of the filter. This means that if you want a time window from Sunday to
  Sunday, `end` must be on Monday.
    
  If `null` this means that no end filter is applied and all entities that are after or on `start`
  are selected.
  """
  end: LocalDate
```

Reply:
This file is now automatically sorted by the GraphQL library, presumably alphabetically. If I revert the order then the test will fail.